lms/djangoapps/instructor/views/api.py

```diff
@@ -2910,3 +2910,98 @@ def generate_certificate_exceptions(request, course_id, generate_for=None):
     }
 
     return JsonResponse(response_payload)
+
+
+USER_INDEX = 0
+NOTES_INDEX = 1
+
+
+@csrf_exempt
+@cache_control(no_cache=True, no_store=True, must_revalidate=True)
+@require_global_staff
+@require_POST
+def generate_bulk_certificate_exceptions(request, course_id):  # pylint: disable=invalid-name
+    """
+    Add Students to certificate white list from the uploaded csv file.
+    """
+    course_key = CourseKey.from_string(course_id)
+    students, warnings, errors, success = ([] for __ in range(4))
```

Comment:
Can we simplify this statement like this 
`students, warnings, errors, success = [], [], [], []`